systemd-user/icyci@.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=icyCI
+
+[Service]
+EnvironmentFile=%h/.config/icyci/%i.conf
+ExecStart=%h/icyci/icyci --source-repo ${SOURCE_REPO} --source-branch ${SOURCE_BRANCH} --test-script ${TEST_SCRIPT} --results-repo ${RESULTS_REPO} --push-source-to-results ${PUSH_SOURCE_TO_RESULTS} --poll-interval ${POLL_INTERVAL}
```

Comment:
Can we drop the `%h/icyci/` path prefix, and assume that icyci is in the user's PATH, or does systemd require an absolute path here?

Reply:
As you discovered in https://github.com/ddiss/icyci/issues/7 , the boolean `--push-source-to-results ${PUSH_SOURCE_TO_RESULTS}` syntax is wrong, so we should probably change to use `--X=Y` syntax for all parameters here too.

Reply:
thanks Michal!

Reply:
As for the PATH resolution -- it'd be evaluated in the environment of `user@.service` which has the system-wide PATH and I'm not sure a regular user can affect it. I.e. relying on PATH would make it require a privileged step in the setup, which is undesired for the user service deployment.